wpimath/src/main/java/edu/wpi/first/math/Drake.java

```diff
@@ -46,13 +47,13 @@ public static SimpleMatrix discreteAlgebraicRiccatiEquation(
    * @param R Input cost matrix.
    * @return Solution of DARE.
    */
-  @SuppressWarnings("ParameterName")
-  public static SimpleMatrix discreteAlgebraicRiccatiEquation(
-          Matrix A,
-          Matrix B,
-          Matrix Q,
-          Matrix R) {
-    return discreteAlgebraicRiccatiEquation(A.getStorage(), B.getStorage(), Q.getStorage(),
-            R.getStorage());
+  @SuppressWarnings({"ParameterName", "MethodTypeParameterName"})
+  public static <States extends Num, Inputs extends Num> Matrix<States, States>
+      discreteAlgebraicRiccatiEquation(Matrix<States, States> A,
+                                   Matrix<States, Inputs> B,
+                                   Matrix<States, States> Q,
+                                   Matrix<Inputs, Inputs> R) {
+    return new Matrix<>(discreteAlgebraicRiccatiEquation(A.getStorage(), B.getStorage(),
+    Q.getStorage(), R.getStorage()));
```

Comment:
Formatting here seems a little off, shouldn't the second line be indented, and the parameters have inconsistent indentation compared to other files (and what was originally in this file).